source/utilities.tex

```diff
@@ -1168,16 +1176,8 @@
 constexpr functions.
 
 \pnum
-The destructor of \tcode{tuple} shall be a trivial destructor if
-\tcode{(is_trivially_destructible_v<Types> \&\& ...)}
-is \tcode{true}.
-
-\pnum
-In the constructor descriptions that follow, let $i$ be in the range
-\range{0}{sizeof...(Types)} in order, $\tcode{T}_i$
-be the $i^\text{th}$ type in \tcode{Types}, and
-$\tcode{U}_i$ be the $i^\text{th}$ type in a template parameter pack named \tcode{UTypes}, where indexing
-is zero-based.
+If \tcode{is_trivially_destructible_v<$\tcode{T}_i$>} is \tcode{true} for all $\tcode{T}_i$
```

Comment:
Add a comma before "then".

Reply:
Done!  I had initially assumed that [variant.dtor] had omitted the comma to avoid any visual collision with the subscript `i`, and so I'd followed the rule "comma if-and-only-if no subscript." I've now changed to "always comma," which I agree is more grammatically correct.